src/System.Security.Cryptography.Primitives/tests/Performance/System.Security.Cryptography.Primitives.Performance.Tests.csproj

```diff
@@ -12,15 +11,11 @@
     <Compile Include="$(CommonTestPath)\System\Security\Cryptography\ByteUtils.cs">
       <Link>CommonTest\System\Security\Cryptography\ByteUtils.cs</Link>
     </Compile>
-    <Compile Include="Perf.FixedTimeEquals.cs" />
```

Comment:
This doesn't seem like a related change.  Why would we no longer want to compile this file?

Reply:
thanks you are right, that was not intended.